### ISSUE TITLE
Not an issue - just wanted to say thanks!!

### ISSUE THREAD

**Issue by averyfreeman** [scopatz/nanorc#148]
Thank you so much for maintaining this branch
I've been using nano on all my *NIX variants for years and the syntax highlighting is so, so great
Please keep the repo up indefinitely!
Thank you - thank you - thank you - thank you!

**Comment by scopatz** [comment]
Much appreciated @averyfreeman! I have no intention of taking this repo down :)

**Comment by DanTheMathsMan** [comment]
I have to agree, finding this has made my life so much easier. You are a great help!

**Comment by ericmjl** [comment]
Also would like to say thanks here! I default to this repo when I setup a new machine :smile:.

**Comment by BarbzYHOOL** [comment]
Me too. I also looked at another similar repo but I dunno, I prefer this one
